backend/util/strings_pt_br.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+"""This file encapsulates all the system's pt-br strings."""
+
+
+STRINGS = {
```

Comment:
You can use this directly from the file, like:

```
from util.strings_pt_br import STRINGS

STRINGS['INVITE_EMAIL_SUBJECT']
```

Reply:
Yes I can, but what if there were many strings into STRINGS that need complement? I would have to import like you said and import the method. If I import only the file, I would call the method with util.strings_pt_br.get_string(). What you think?